index.html

```diff
@@ -2614,6 +2639,90 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double outputLatency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the
+              interval between the time the UA requests the host system to play
+              a buffer and the time at which the first sample in the buffer is
+              actually processed by the audio output device. For audible
+              devices such as speakers or headphones that produce an acoustic
+              signal, this latter time refers to the time when a sample's sound
+              is produced.
+            </p>
+            <p>
+              The <a><code>outputLatency</code></a> attribute value depends on
+              the platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>outputLatency</code></a> attribute value
+              will be updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimeStamp outputTimeStamp
+          </dt>
+          <dd>
+            <p>
+              Holds an <a><code>AudioTimeStamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a href=
+              "#widl-AudioTimeStamp-contextTime"><code>contextTime</code></a>
```

Comment:
@rtoy In practice the difference can be seen at https://github.com/WebAudio/web-audio-api/issues/12#issuecomment-200521873 the `AudioTimeStamp` values are always behind `context.currentTime` and `performance.now()`. The difference depends on the system buffer size and output latency value.

`context.currentTime` represents the sample frame immediately following the last sample-frame in the block of audio most recently processed by the context's rendering graph.

`AudioTimeStamp.contextTime` represents the sample frame which is currently being rendered by the speaker. It can be considered as an analogue of [IAudioClock::GetPosition](https://msdn.microsoft.com/en-us/library/windows/desktop/dd370889(v=vs.85).aspx) from WASAPI.